### DISCUSSION TITLE
Search first mode

### DISCUSSION THREAD

**Discussion by jrich523** [VueTorrent/VueTorrent#1980] [Ideas]
I love this project, thank you!

I find that the way i use this the most is from my phone and when i say "oh i should get X" and i go to search to find and download something. 

The search is an overlay/frame it seems and using that on the phone is a little rough (have to turn my phone sideways to get it to be usable/download)

I dont really care about active downloads since 90% of them just work and are pretty quick. 

It would be super cool if there was a mode that made search the primary view/interface.

again, really love this project, thank you!

**Comment by Larsluph** [comment]
Unfortunately making search engine the primary element looks like something out of scope for VueTorrent. I think it would make more sense to have another WebUI focusing on that particular use case instead.

However the fact that the search engine view is "unusable" on mobile is something we can look at, can you send some examples / screenshots of problems you encounter?

**Reply by jrich523** [reply]
that makes sense. i just pulled the most recent version and it seems to be more functional on the phone since i last bothered to use it in portrait. the two issues that made me rotate my phone was the sort by seeds (having the ability to set a default sort type, or to reuse the last would be nice)

the other issue was actually triggering the download.

all in all the portrait view shows cards where the landscape view shows a table, and the table being way easier to use. not totally sure how you could make the portrait view as quick/easy as the table view of the landscape.

i believe there is no customization of the display of search results?

again, love this project, amazing work! thanks!

**Reply by Larsluph** [reply]
This is Vuetify's choice of using a card layout for mobile, because tables aren't considered responsive in the general web consortium, because mobile is a vertical-focused layout compared to the horizontal pattern of a table.

Designing our own mobile layout is possible so don't hesitate if you have any ideas of the design we can use for this.

**Reply by jrich523** [reply]
I guess depending on how much control over the cards you have, maybe a more compact view might help

![image](https://github.com/user-attachments/assets/46f6e3ef-1f1d-4dc5-bede-51a1d7b3bc3a)

left is current, right would be a more compact way to view it would would likely take it from just over 1 card , to 3.
the left side is all i can see for results on my pixel 7 pro

**Reply by Larsluph** [reply]
I just opened a PR to make your suggested layout: #2059

**Reply by jrich523** [reply]
super cool, thank you!

**Reply by jrich523** [reply]
hey @Larsluph!

Love this update, thank you.
I did however notice that the `Sort by` dropdown now shows the data of the cards. I was going to attach a screenshot but its super hard to see, it just looks like the cards.

**Reply by Larsluph** [reply]
It's a known Vuetify bug, see #2089
